### ISSUE TITLE
Broken layout in IE11

### ISSUE THREAD

**Issue by davejamesmiller** [lemonmade/bigfoot#36]
In IE11 the popup is not positioned correctly:

![2015-03-22 08_33_46](https://cloud.githubusercontent.com/assets/236616/6768647/a3f02f42-d070-11e4-84d6-c132670ecfdd.png)

I tracked it down to `window.scrollX`/`Y` in `viewportDetails()` - these are null in IE11. That causes `roomCalc()` to return `NaN` for all values.

I fixed it by replacing that function in `bigfoot.js` with this, which uses jQuery to get the values:

```js
      viewportDetails = function() {
      	var $window = $(window);

        return {
          width: window.innerWidth,
          height: window.innerHeight,
          scrollX: $window.scrollLeft(),
          scrollY: $window.scrollTop()
        };
      };
```

**Comment by lemonmade** [comment]
Thanks for the report — fix is in 033a2508e2c32a6aec38de7de01b13bce380eae4.

**Comment by davejamesmiller** [comment]
Thank you! :+1:
